textbaseRPG/Class.java

```diff
@@ -0,0 +1,79 @@
+public class Class{
+        static String classesType;
+        static String strength;
+        static String wisdom;
+        static String agility;
+        static String high = "High";
+        static String avg = "Average";
+        static String low = "Low";
+        //does this work?
+
+
+    public String[] classes(){
+        String[] classes = {"Knight", "Wizard", "Rogue", "Swordsman"};
+        return classes;
+        //not sure what to do here yet
+    }
+
+    public static void knight(){
+        strength = high;
+        wisdom = avg;
+        agility = low;
+    }
+    public static void wizard(){
+        strength = low;
+        wisdom = "Very High";
+        agility = low;
+    }
+    public static void rogue(){
+        strength = avg;
+        wisdom = low;
+        agility = high;
+    }
+    public static void swordsman(){
+        strength = avg;
+        wisdom = avg;
+        agility = avg;
+    }
+
```

Comment:
Why do we need this? We do not need to assign avg, high ,low to strength stat

Reply:
how about we create another class called DamgeCounter that will multiply this avg,high,low setting? So when we set high,low, avg, we can return a static int in the field. This will be used to calculate damage later on.